src/ocean/math/IEEE.d

```diff
@@ -275,28 +275,32 @@ private:
         }
     }
 private:
-    static IeeeFlags getIeeeFlags()
+    static int getIeeeFlags()
```

Comment:
Shouldn't we be able to replace `int` with `IeeeFlags` again now that we have `naked` and `ret` in the asm?